client/extensions/wp-super-cache/wrap-settings-form.jsx

```diff
@@ -50,16 +93,22 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 				handleToggle: this.handleToggle,
 			};
 
-			return <SettingsForm { ...this.props } { ...utils } />;
+			return (
+				<div>
+					<QuerySettings siteId={ this.props.siteId } />
```

Comment:
@dbtlr This is a type of component that Calypso calls a [query component](https://wpcalypso.wordpress.com/devdocs/docs/our-approach-to-data.md#query-components). It's what kicks off the process of fetching the WP Super Cache settings.